website/content/docs/api-clients/client-agent.mdx

```diff
@@ -277,6 +277,14 @@ If you want to disable the Boundary Client Agent, you can stop it with the follo
 $ sudo launchctl stop com.hashicorp.boundary.boundary-client-agent
 ```
 
+<Note>
+
+Stopping the Boundary client agent using `launchctl stop` only halts it temporarily, as the `StartInterval` setting causes it to restart automatically after 60 seconds.
+
+**To disable it permanently**, you must unload the service using `launchctl unload`. Unloading not only stops the agent but also removes its configuration from `launchd`. To restart the agent after unloading, you must reload it using `launchctl load`, as the `start` command will not work in this case.
+
+</Note>
```

Comment:
Thanks for writing this, however, I think we should just include the `unload` command in the above shell session to avoid confusion. Is both `stop` and `unload` required, or just `unload`?

Reply:
Makes sense, adding the `unload` command directly will make the instructions clearer and avoid confusion.  

On the other hand, there might be a couple of minor points that could potentially cause confusion:  
1. Users familiar with `launchd` might use `stop`, not realizing it restarts after 60 seconds due to the `StartInterval` setting.  
2. Users who unload the service might not know they need to reload it with `launchctl load` to start it again, as this behavior isn’t explicitly mentioned.  

These might not be significant issues, but do you think it’s worth briefly addressing them?

Reply:
Yeah, I think we could have the shell command say how to unload it, and then have a note to say how to start it again. Do you mind adding the equivalent for Windows too?

Reply:
For Windows, there isn’t an equivalent to `StartInterval` like on macOS. Once the BoundaryClientAgent is stopped using `net stop BoundaryClientAgent`, it remains stopped permanently unless explicitly started again. Therefore, there’s no need to update anything specific for Windows. The `net stop BoundaryClientAgent` command works perfectly fine as expected.